docs/operations/containerized_data_importer.md

```diff
@@ -21,7 +21,8 @@ virtctl in your path.
 Install the latest CDI release
 [here](https://github.com/kubevirt/containerized-data-importer/releases)
 
-    VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
+    export TAG=$(curl -s -w %{redirect_url} https://github.com/kubevirt/containerized-data-importer/releases/latest)
```

Comment:
Note this is the bash function we use in the code to get the latest you just call `get_latest_release "kubevirt/kubevirt" and it finds the latest release.

```
get_latest_release() {
  curl -s "https://api.github.com/repos/$1/releases/latest" |       # Get latest release from GitHub api
    grep '"tag_name":' |                                            # Get tag line
    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value (avoid jq)
}
```
Now I don't mind your version, but it might make sense to show the same as the actual project.

Reply:
@awels we use the above curl | grep -o structure in multiple places in the docs (and labs, and scenarios), I really like the %{redirect_url} trick, and would consider that more future safe than a grep of a generated page...

Once we decide which of the two methodologies is best, @rafaeldomi would you be open to creating a similar PR against https://github.com/kubevirt/kubevirt.github.io ?

Reply:
Also, a one-liner way (if needed) would be to nest $() executions like:

```
export VERSION=$(basename $(curl -s -w %{redirect_url} https://github.com/kubevirt/containerized-data-importer/releases/latest))
```

Reply:
Note I am perfectly fine with what is shown here.

Reply:
> Note I am perfectly fine with what is shown here.

As am I

/approve